### DISCUSSION TITLE
Edit client config file

### DISCUSSION THREAD

**Discussion by tobias-carlbom** [wg-easy/wg-easy#1326] [Q&A]
Hi,

Below is a downloaded client config file downloaded from wg-easy. I want to be able to permanently edit the AllowedIPs. Seems that i cannot manupilate this in the web ui, or am i missing anything?

I am running it as a docker container, and i checked in the cli under /etc/wireguard, but i only found wg0.conf

`
[Interface]
PrivateKey = iLdLBrjzgXK810Y0fXlQa2CFq1MpMrhJ7hQ+TVy4BGs=
Address = 10.8.0.2/24
DNS = 1.1.1.1

[Peer]
PublicKey = sHqZ2zYShxL6D46OSAaR0Xy4pvv5fZSItNz0RbUuuXM=
PresharedKey = why0pQfYq/4yhVJVwCwuLuJ3PuzF2rSYDHVemUW31cs=
AllowedIPs = 0.0.0.0/0, ::/0
PersistentKeepalive = 0
Endpoint = casaos.local:51820
`

**Comment by eandersons** [comment]
Are you looking for something like `WG_ALLOWED_IPS`:

> Allowed IPs clients will use.

Note that this environment variable will affect only newly created peers and users can change this value manually on their devices. For example, on Android WireGuard app:
![image](https://github.com/user-attachments/assets/c32c95f3-b1bd-4fcb-ad5e-60c4d64d4eae)

If you want more permanent restrictions regarding allowed IPs, it probably can be achieved with some ip_tables trickery, but I am not sure and I do not know how.
